core/db/migrate/20230111122511_transfer_product_and_taxon_data_to_translatable_tables.rb

```diff
@@ -0,0 +1,80 @@
+class TransferProductAndTaxonDataToTranslatableTables < ActiveRecord::Migration[6.1]
+  DEFAULT_LOCALE = 'en'
+  PRODUCTS_TABLE = 'spree_products'
+  PRODUCT_TRANSLATIONS_TABLE = 'spree_product_translations'
+  TAXONS_TABLE = 'spree_taxons'
+  TAXON_TRANSLATIONS_TABLE = 'spree_taxon_translations'
+
+  def up
+    # Only transfer data if translation tables are being newly created / no translations exist
+    # Otherwise, assume translation data is already in place from spree_globalize
+
+    # Products
+    if not Spree::Product::Translation.exists?
```

Comment:
There's a table_exists? method available, which allows us to make it more explicit
```suggestion
    unless table_exists?(PRODUCT_TRANSLATIONS_TABLE)
```

Reply:
Ah sorry, I misunderstood this migration. Please disregard my comment :)